webcompat/form.py

```diff
@@ -41,19 +45,16 @@
 
 class IssueForm(Form):
     '''Define form fields and validation for our bug reporting form.'''
-    url = StringField(u'Site URL*', [Required(message=url_message)])
+    url = StringField(u'Site URL*', [InputRequired(message=url_message)])
     browser = StringField(u'Browser / Version', [Optional()])
     os = StringField(u'Operating System', [Optional()])
-    summary = StringField(u'Problem in 5 words*',
-                          [Required(message=summary_message)])
     username = StringField(u'Username',
                            [Length(max=0, message=username_message)])
-    description = TextAreaField(u'How can we replicate this?', [Optional()],
+    description = TextAreaField(u'Give more details', [Optional()],
                                 default=desc_default)
-    site_owner = RadioField(u'Is this your website?', [Optional()],
-                            choices=owner_choices)
-    problem_category = RadioField(u'What seems to be the trouble?',
-                                  [Optional()], choices=problem_choices)
+    problem_category = RadioField(u'What seems to be the trouble?*',
```

Comment:
Nah, we have the `*` to indicate required (Currently on our Problem in 5 words* and Site URL* labels as wel. But now I notice we don't explain the * anywhere (maybe we don't need to?)